lib/rules/max-lines-per-function.js

```diff
@@ -86,6 +89,7 @@ module.exports = {
         let maxLines = 50;
         let ignoreComments = true;
         let skipBlankLines = true;
+        let ignoreIIFEs = true;
```

Comment:
Why is a boolean defaulted to true?

If the default is to be to ignore IIFEs (which i don’t agree with) it should be named such that the default is false. Perhaps just “IIFEs”?

Reply:
OK, I have changed the defaults for the boolean args to default to false (just like rule max-len does) and renamed the option to "IIFEs"